src/componentprocessor/index.ts

```diff
@@ -19,6 +19,12 @@ interface ComponentField {
     [key: string]: FieldAttributes;
 }
 
+interface ComponentNode {
+    name: string;
+    fields: Object;
```

Comment:
Is there a way we could use a more informative type than `Object` here?

Reply:
It would be something like this:
```
interface ComponentField {
  [key: string]: [value: string];
}
```
but don't think this is pretty useful. I'll give a try later